llvmlite/tests/test_ir.py

```diff
@@ -860,6 +861,22 @@ def test_structs(self):
         c = ir.Constant(ir.LiteralStructType((flt, int1)), ir.Undefined)
         self.assertEqual(str(c), '{float, i1} undef')
 
+    def test_encoding_problem(self):
+        celems = [ir.Constant(ir.ArrayType(ir.IntType(8), 8),
```

Comment:
Does it have to be this complicated? Why not a single constant with e.g. `bytearray(b"\x00\xff")` ?